pkg/agent/manager/cache/cache_test.go

```diff
@@ -689,6 +842,36 @@ func BenchmarkCacheGlobalNotification(b *testing.B) {
 	}
 }
 
+func newTestCache() *Cache {
+	log, _ := test.NewNullLogger()
+	return New(log, spiffeid.RequireTrustDomainFromString("domain.test"), bundleV1,
+		telemetry.Blackhole{}, 0, 0, clock.NewMock())
+}
+
+func newTestCacheWithConfig(maxSvidCacheSize int, svidCacheExpiryPeriod time.Duration, clk clock.Clock) *Cache {
+	log, _ := test.NewNullLogger()
+	return New(log, spiffeid.RequireTrustDomainFromString("domain.test"), bundleV1, telemetry.Blackhole{},
+		maxSvidCacheSize, svidCacheExpiryPeriod, clk)
+}
+
+func createUpdateEntries(numEntries int, bundles map[spiffeid.TrustDomain]*bundleutil.Bundle) *UpdateEntries {
+	updateEntries := &UpdateEntries{
+		Bundles:             bundles,
+		RegistrationEntries: make(map[string]*common.RegistrationEntry, numEntries),
+	}
+
+	for i := 0; i < numEntries; i++ {
+		entryID := fmt.Sprintf("00000000-0000-0000-0000-%012d", i)
```

Comment:
maybe specify a limit in comments (or fail test) if `numEntries` is too large and thus will lead to dupe reg IDs?

Reply:
Added comment but we would not create these many entries in unit tests :)